components/meta-packages/history/history

```diff
@@ -1021,6 +1021,7 @@ system/library/dbus-x11@1.12.8,5.11-2018.0.0.1 system/library/dbus/dbus-x11@1.12
 system/library/libvirt@0.5.11,5.11-2015.0.2.0
 system/library/sysidtool@0.5.11,5.11-2015.0.2.0
 system/library/security/crypto/pkcs11_kms@0.5.11,5.11-2015.0.2.0
+system/library/usb/openusb@1.0.1-2018.0.0.2
```

Comment:
SUNWlibopenusb should also be obsolete, not renamed (can't rename package to obsolete package).

Reply:
I fixed it, thanks.